internal/dispatch/graph/graph.go

```diff
@@ -198,13 +199,21 @@ func (ld *localDispatcher) DispatchLookup(ctx context.Context, req *v1.DispatchL
 
 	if ld.requiresLookupViaChecks(relation) {
 		log.Warn().Msg("slow path dispatch lookup")
-		slowLookupCounter.Inc()
+		slowLookupCounter.WithLabelValues(prefix(validatedReq.Subject.Namespace)).Inc()
 		return ld.lookupHandler.LookupViaChecks(ctx, validatedReq)
 	}
 
 	return ld.lookupHandler.Lookup(ctx, validatedReq)
 }
 
+func prefix(namespace string) (prefix string) {
```

Comment:
We must have some code for this somewhere...

Reply:
I didn't see anything that assumed the namespace string had already been validated like it is here